sympy/logic/boolalg.py

```diff
@@ -136,6 +100,53 @@ class BooleanTrue(with_metaclass(Singleton, BooleanAtom)):
     True they act bitwise on 1. Functions in the logic module will return this
     class when they evaluate to true.
 
+    Notes
+    =====
+
+    There is liable to be some confusion as to when `True` should
+    be used and when `S.true` should be used in various contexts
+    throughout SymPy. An important thing to remember is that
+    sympify(True) returns `S.true`. This means that for the most
+    part, you can just use `True` and it will automatically be converted
+    to `S.true` when necessary, similar to how you can generally use 1
+    instead of S.One.
+
+    The rule of thumb is:
+
+    "If the boolean in question can be replaced by an arbitrary symbolic
+    Boolean, like Or(x, y) or x > 1, use `S.true`.
+    Otherwise, use `True`"
+
+    In other words, use `S.true` only on those contexts where the
+    boolean is being used as a symbolic representation of truth.
+    For example, if the object ends up in the .args of any expression,
+    then it must necessarily be `S.true` instead of `True`, as
+    elements of .args must be Basic. On the other hand,
+    == is not a symbolic operation in SymPy, since it always returns
+    True or False, and does so in terms of structural equality
+    rather than mathematical, so it should return True. The assumptions
+    system should use True and False. Aside from not satisfying
```

Comment:
... etc. ... - anything that's a code snippet can be enclosed in backticks.
BTW I'm just in "oh while we're at it, it would be nice to have this as well" mode, I'm not insisting. Feel free to say "I don't have the time to fix this as well".